jetty-deploy/src/main/config/etc/global-webapp-deploy.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<!--
+  ~ Copyright 2016 Google Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_0.dtd">
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Ref refid="DeploymentManager">
+    <Call name="addLifeCycleBinding">
+      <Arg>
+        <New class="org.eclipse.jetty.deploy.bindings.GlobalWebappConfigBinding">
+          <Set name="jettyXml">
+            <Property>
+              <Name>jetty.deploy.globalWebappConfig</Name>
+              <Default>
+                <Property name="jetty.base"/>/etc/global-webapp-config.xml
```

Comment:
A template `global-webapp-config.xml` should be created in `jetty-deploy/src/main/config/modules/globalwebapp.d/global-webapp-config.xml` and copied into jetty-base/etc by the module activation.  In fact `global-webapp-deploy.xml` should also be copied into place in the same way (ie don't pollute etc with config files that are not used).

Reply:
I changed the name of the module to deployer-webapp-common, and the xml file for the Deployer config to be deployer-webapp-common.xml and added an empty config file for the webapp called webapp-common.xml. Both are copied into $jetty.base/etc.

Not sure I agree with the practice of copying $jetty.home xml files that should never be modified into $jetty.base/etc, as this just encourages users to modify them and then it makes upgrades more complicated.